k8s/deploy_alasybil.sh

```diff
@@ -1,9 +0,0 @@
-#!/bin/bash
```

Comment:
could we keep this script with the registry updated? i find it very useful for now.